articles/cognitive-services/openai/how-to/chatgpt.md

```diff
@@ -25,12 +25,12 @@ The following code snippet shows the most basic way to use the ChatGPT model. We
 import os
 import openai
 openai.api_type = "azure"
-openai.api_base = "https://{your-resource-name}.openai.azure.com/"
+openai.api_base = "https://{your-resource-location}.api.cognitive.microsoft.com/"
```

Comment:
Unfortunately, this suggestion is incorrect. For an Azure OpenAI resource the endpoint follows the pattern: 

`https://{your-resource-name}.openai.azure.com/`

location is not part of the endpoint address.

This is unique to Azure OpenAI resources, other Cognitive Services resources follow a different pattern.